src/theme/Tabs/index.js

```diff
@@ -78,7 +78,6 @@ function Tabs(props) {
     <div>
       <ul
         role="tablist"
-        aria-orientation="horizontal"
```

Comment:
`aria-orientation="vertical"`

Reply:
@jjwill It alternates between vertical and horizontal based on screen width. I think I'll create a ticket for it to deal with that later